lms/tasks/celery.py

```diff
@@ -38,6 +38,9 @@
         "interval_start": 0.2,
         "interval_step": 0.2,
     },
+    task_routes={
+        "lms.tasks.email_digests.*": "email_digests",
```

Comment:
We already namespace our tasks so this works nicely with the `*`

Reply:
:+1:

This is probably a good example of the kind of thing--if anyone every renames `lms/tasks/email_digests.py`--a simple grip for `email_digests` would find the reference in this string but most smarter refactoring tools wouldn't